### PR TITLE
Deadmau6 version control

### LINKED ISSUE #42: Setup Versioning and Changelog
For versioning I think we should follow [semver](https://semver.org/) standards.

### PR DESCRIPTION
**Pull request by deadmau6**
Fixes issue #42

### CODE REVIEW

**Review by connorjoleary — COMMENTED** [review]
Does this affect the version of the model, lambda, and extension? Or is this a separate version?

**Comment by deadmau6** [comment]
> Does this affect the version of the model, lambda, and extension? Or is this a separate version?

So currently this would affect `VERSIONS_EXTENSION` but I could include the `API`, `MODEL`, and `LAMBDA` to be under a single `VERSION`. Is that something that we want?

**Comment by connorjoleary** [comment]
For this project, I am thinking of using a micro-service architecture, with the model, extension, and lambda all being a different service. What I am reading is suggesting that each service should have its own version, so I dont think that we should group them into one.

**Comment by connorjoleary** [comment]
But the change log you are suggesting would only apply to the extension right?

**Comment by deadmau6** [comment]
> But the change log you are suggesting would only apply to the extension right?

Correct and if we are doing a `microservices` architecture I agree we should have separate versions and `CHANGELOGS`, so I will get on that. I am gonna be out of town this weekend so I will have to get back to this next week hope that's okay?

**Comment by connorjoleary** [comment]
Yes! And thank you so much for putting this together.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# DeepCite
+# DeepCite ![version](https://img.shields.io/badge/version-2.0.0-blue)
```
Comment (connorjoleary) [inline README.md]: Could we hold off on a 2.0.0 version? In my head we are not even at 1.0.0. Also this badge is just for the extension?

File: scripts/backend_versions.py [hunk 2]
```diff
@@ -0,0 +1,154 @@
+#!/bin/python3
+import argparse
+import json
+import os
+
+DEFAULT_FPATH = os.path.join(os.getcwd(), '..', 'backend', 'aws', 'lambda', 'defaults.json') 
+PACKAGE_FPATH = os.path.join(os.getcwd(), '..', 'extension','package.json')
+MANIFEST_FPATH = os.path.join(os.getcwd(), '..', 'extension', 'manifest.json')
```
Comment (connorjoleary) [inline scripts/backend_versions.py:8]: Thank you for adding these in!

**Review by connorjoleary — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (deadmau6) [reply README.md]: yea it is just for the extension I can specify that, and even add the others

**Pull request merged**